File: ExcAlgoritmoProgramacao_JS/Lista_Bee/1134_basico.js
import { question } from "readline-sync";

function main() {
    let total_gasolina = 0;
    let total_alcool = 0;
    let total_diesel = 0;

    while (true) {
        let voto = get_number();
        if (voto === 1 || voto === 2 || voto === 3) {
            voto = (voto === 1) ? total_alcool += 1 : (voto === 2) ? total_gasolina += 1 : total_diesel += 1;
        } else if (voto === 4) {
            break;
        }
    }
    print(`MUITO OBRIGADO`);
    print(`Alcool: ${total_alcool}`);
    print(`Gasolina: ${total_gasolina}`);
    print(`Diesel: ${total_diesel}`);
}

function get_number() {
    return parseInt(question());
}

function print(texto) {
    return console.log(texto);
}

main();

File: Exercicios_ED_I/Bee/1134_basico.cpp
#include <stdio.h>
#include <iostream>
using namespace std;

main() {
    int quant_Gaso = 0, quant_Alco = 0, quant_Dies = 0, combustivel;
    cin >> combustivel;

    while (combustivel != 4) {
        while (combustivel < 1 || combustivel > 4) {
            cin >> combustivel;
        }
        
        if (combustivel == 1) {
            quant_Alco++;
        } else if (combustivel == 2) {
            quant_Gaso++;
        } else if (combustivel == 3){
            quant_Dies++;
        }
        cin >> combustivel;
    }
    cout << "MUITO OBRIGADO" << endl;
    cout << "Alcool: " << quant_Alco << endl;
    cout << "Gasolina: " << quant_Gaso << endl;
    cout << "Diesel: " << quant_Dies << endl;
}